collections/_posts/2023-09-28-PLIP.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: review
+title: "A visual–language foundation model for pathology image analysis using medical Twitter"
+tags: CLIP Foundation-Models Pathology-Images
+author: "Hang Jung Ling"
+cite:
+    authors: "Zhi Huang, Federico Bianchi, Mert Yuksekgonul, Thomas Montine, James Zou"
+    title:   "A visual–language foundation model for pathology image analysis using medical Twitter"
+    venue:   "Nature Medicine"
+pdf: "https://www.biorxiv.org/content/10.1101/2023.03.29.534834.full"
+---
+
+# Notes
+* Official demo website (including links to the pretrained model, GitHub code, and dataset) is available on Hugging Face via this [link](https://huggingface.co/spaces/vinid/webplip).
+* Published in Nature Medicine (IF: 87) > Nature (IF: 70). 😉
+
+# Highlights
+- Introduce a large dataset (~200k) of pathology images with natural language descriptions.
+- Introduce the first pre-trained foundation model (fine-tuned from pre-trained CLIP[^1]) specialized for pathology images with both image and text understanding, named as pathology language–image pretraining (PLIP).
+
+# Introduction
+Limitation of AI-enhanced diagnoses from routinely stained hematoxylin and eosin (H&E) images:
+- lack of diversified datasets that include well-annotated labels in natural language considering that there are more than 8,000 diseases.
+
+At the same time, many de-identified pathology images are shared on the Internet, especially on social media (e.g., Twitter), where clinicians discuss de-identified medical images with their colleagues.
+
+# Creation of OpenPath dataset
+![](/collections/images/PLIP/dataset_creation.jpg)
+***a,** Flowchart of data acquisition from medical Twitter. **b,** Overview of the OpenPath dataset. **c,** Total number of available image–text pairs from tweets and replies within each Twitter hashtag (sorted in alphabetical order). Replies are those that received the highest number of likes in Twitter posts, if applicable. **d,** Density plot of the number of words per sentence in the OpenPath dataset.*
```

Comment:
I'd say that normally the caption for each letter in the figure would use parenthesis rather than commas:
```suggestion
***a)** Flowchart of data acquisition from medical Twitter. **b)** Overview of the OpenPath dataset. **c)** Total number of available image–text pairs from tweets and replies within each Twitter hashtag (sorted in alphabetical order). Replies are those that received the highest number of likes in Twitter posts, if applicable. **d)** Density plot of the number of words per sentence in the OpenPath dataset.*
```